alerts/cluster-monitoring-operator/KubeStateMetricsShardingMismatch.md

```diff
@@ -0,0 +1,31 @@
+# KubeStateMetricsShardingMismatch
```

Comment:
we should drop this one. I think it's been included in a recent kube-prometheus bump but we don't do KSM sharding so this alert isn't relevant.